primitives/src/vid/advz/payload_prover.rs

```diff
@@ -237,7 +232,6 @@ where
             suffix_elems,
             prefix_bytes: payload[range_elem_byte.start..range.start].to_vec(),
             suffix_bytes: payload[range.end..range_elem_byte.end].to_vec(),
-            chunk_range: range,
         })
     }
 
```

Comment:
Should we rename the file `payload_prover` if it also contains verification APIs?

Reply:
I'm open to that but I can't think of a name I like more. `payload_prover_and_verifier.rs` is too long. Any suggestion? I thought maybe "prover" in the name suggests that there's also a verifier included, but I admit that's not necessarily obvious.

I have another PR coming with a naming challenge of its own. Perhaps it makes sense to do a rename in the next PR. I'll merge now that I have approval.